target/linux/meson/Makefile

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright (C) 2017-2021 Stijn Tintel <stijn@linux-ipv6.be>
```

Comment:
this copyright doesn't seem to belong to you, replace it with your own one, or just drop it